models.py

```diff
@@ -201,11 +226,13 @@ def forward(self, x, r, embed=True):
 class ByteCNNDecoder(nn.Module):
     def __init__(self, n, emsize, vocab_size, normalization,
                  use_linear_layers=True, compress_channels=None,
-                 output_embeddings_init=None):
+                 output_embeddings_init=None,
+                 use_external_batch_norm=False, external_batch_max_r=None):
```

Comment:
W dekoderze to chyba nie ma sensu, chyba, że chcesz zrobić wersję "kwadratową".

Reply:
W rzeczy samej, wersja kwadratowa.